app/gui/src/ide/integration/project.rs

```diff
@@ -776,69 +777,76 @@ impl Model {
     /// Reload whole displayed content to be up to date with module state.
     pub fn refresh_graph_view(&self) -> FallibleResult {
         info!(self.logger, "Refreshing the graph view.");
-        let connections_info = self.graph.connections()?;
-        self.refresh_node_views(&connections_info, true)?;
-        self.refresh_connection_views(connections_info.connections)?;
-        Ok(())
+        profiling::measure_task!("refresh_graph_view", || {
+            let connections_info = self.graph.connections()?;
+            self.refresh_node_views(&connections_info, true)?;
+            self.refresh_connection_views(connections_info.connections)?;
+            Ok(())
+        })
```

Comment:
A small note – I'm not sure if its good idea to keep the syntax `|| { ... }` here – it suggest to the reader that this is a lambda, where `?` syntax should work differently inside. Maybe we should just use `{ ... }` syntax here instead?